app/upgrades.go

```diff
@@ -191,6 +193,16 @@ func (app *StrideApp) setupUpgradeHandlers(appOpts servertypes.AppOptions) {
 		),
 	)
 
+	// v15 upgrade handler
+	app.UpgradeKeeper.SetUpgradeHandler(
```

Comment:
Checking - the variable that Riley is setting doesn't need a new store, right? (we don't need to init a new store key)

Reply:
we shouldn't need a new store key cause it's part of the stakeibc store. would be nice to confirm if possible though